apps/prairielearn/src/ee/lib/aiQuestionGeneration.ts

```diff
@@ -148,10 +149,32 @@ Keep in mind you are not just generating an example; you are generating an actua
     });
 
     extractFromCompletion(completion, job);
+    const html = job?.data?.html;
+
+    job.data['initialGenerationErrors'] = [];
     job.data['prompt'] = prompt;
     job.data['generation'] = completion.choices[0].message.content;
     job.data['context'] = context;
     job.data['completion'] = completion;
+
+    if (html && typeof html === 'string') {
```

Comment:
What if there _is_ no HTML? Presumably we want to handle that, possibly via regeneration and a "you need to generate some HTML" instruction?

Reply:
Changed in c983452c1d39437984380e232edd10e602a5c8a0.